packages/terra-clinical-result/src/terra-dev-site/test/clinical-result/result-name-header-cell/SemanticTableWithResultNameHeaderCell.test.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { ResultNameHeaderCell } from 'terra-clinical-result/lib/index';
+import Table, {
+  Header,
+  Row,
+} from 'terra-html-table';
+
+const SemanticTableWithResultNameHeaderCell = () => (
+  <Table>
+    <Header>
+      <Row>
+        <ResultNameHeaderCell resultName="Temp" unit="degC" />
+      </Row>
+      <Row>
+        <ResultNameHeaderCell resultName="BP" unit="mmHg" />
+      </Row>
+      <Row>
+        <ResultNameHeaderCell resultName="HR" unit="mmHg" />
+      </Row>
+    </Header>
+  </Table>
+);
```

Comment:
Looking at this new ResultNameHeaderCell example, is it possible to keep the name and unit at the same level(like the example in the component site)?
<img width="1673" alt="Screenshot 2023-06-09 at 2 12 46 PM" src="https://github.com/cerner/terra-clinical/assets/44185348/b43566e5-1657-4b21-8822-4fdee4c5e331">

Reply:
Good catch! Updated the css here: [002e458](https://github.com/cerner/terra-clinical/pull/865/commits/002e458494d9942bd483eb5904dc2bb8a411138e)

The tests are still running but here's what it looks like now:
<img width="623" alt="Screenshot 2023-06-12 at 11 26 20 AM" src="https://github.com/cerner/terra-clinical/assets/40574651/18843735-9e1e-4835-9e02-9acb9eb4d448">